### DISCUSSION TITLE
Full portable mode?

### DISCUSSION THREAD

**Discussion by ghost** [VSCodium/vscodium#1649] [Ideas]
I'm needing an install of VSCodium on a USB stick as I move from PC to PC a lot, but even with the standalone exe, the config for the app like extensions and settings don't come with me. Could an option for a local config folder be considered? So the config will be carried from system to system along with the app itself.

**Comment by githubyouser** [comment]
Looks like this already supported! See here: https://github.com/VSCodium/vscodium/blob/master/docs/index.md#portable

**Comment by ni6hant** [comment]
> After unzipping the VS Code download, create a data folder within VS Code's folder:
> Text
> 
> |- VSCode-win32-x64-1.84.2
> |   |- Code.exe (or code executable)
> |   |- data
> |   |- bin
> |   |  |- code
> |   |  |- ...
> |   |- ...
> 
> From then on, the data folder will be used to contain all VS Code data, including session state, preferences, extensions, etc.
> Note
> 
> The data folder will override the --user-data-dir and --extensions-dir [command line](https://code.visualstudio.com/docs/configure/command-line#_advanced-cli-options) options.
> 
> The data folder can be moved to other VS Code installations. This is useful for updating your portable VS Code version, in which case you can move the data folder to a newer extracted version of VS Code.

From: https://code.visualstudio.com/docs/editor/portable
